browser/ethereum_remote_client/ethereum_remote_client_delegate_impl.cc

```diff
@@ -27,3 +27,16 @@ void EthereumRemoteClientDelegateImpl::MaybeLoadCryptoWalletsExtension(
   }
 #endif  // #if BUILDFLAG(ENABLE_EXTENSIONS)
 }
+
+void EthereumRemoteClientDelegateImpl::RemoveCryptoWalletExtension(
+    content::BrowserContext* context) {
+#if BUILDFLAG(ENABLE_EXTENSIONS)
```

Comment:
`#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED) && BUILDFLAG(ENABLE_EXTENSIONS)` from 
`#include "brave/browser/ethereum_remote_client/buildflags/buildflags.h"`

Reply:
it is already `assert(ethereum_remote_client_enabled)` in the BUILD.gn that has `ethereum_remote_client_delegate_impl.cc` and `ethereum_remote_client_delegate_impl.h`

Reply:
If you want you can probably just make `ethereum_remote_client_enabled = !is_ios && !is_android` chagned to `enable_extensions` and then not have any check for extensions there at all

Reply:
addressed in fixed in 615ec530a09034306bb5efdea47492e0464e5558